BRB/run.py

```diff
@@ -27,15 +28,21 @@
    help='specify a custom ini file.',
    show_default=True
 )
-def run_brb(configfile):
+@click.option(
+    "-s",
+    "--stats",
+    required=False,
+    help='specify a flowcell ID to push its stats.'
+)
+def run_brb(configfile, stats):
 
     while True:
-        #Read the config file
+        # Read the config file
         config = BRB.getConfig.getConfig(configfile)
-
-        #Get the next flow cell to process, or sleep
+        
+        # Get the next flow cell to process, or sleep
         config, ParkourDict = BRB.findFinishedFlowCells.newFlowCell(config)
-        if(config.get('Options','runID') == '') or ParkourDict is None:
+        if(config.get('Options','runID') == '') or ParkourDict is None or stats is None:
```

Comment:
If I read this correctly you will always hit the sleep arm if --stats is not specified ?

Reply:
Yes, the idea here is that we get into the loop with a new behavior (stats enabled), and jump directly to pushing stats and then finish the process. Meanwhile, stats disabled, should operate as usual (looking for flowcells, loading them into the config object, and with the sleep)

Seems a bit hacky, but I couldn't think of anything else. Well, the alternative would be simply writing a python script, independent to BRB, that takes FCID and pushes stats. I might go with this if the current approach fails and turns to be rather cumbersome... Admittedly, I'm not proud of how things look right now. And it's not just being a draft... but it really has this "stats is not None" here and there, to use the same loop... :/